releasenotes/notes/add-rocky-9.3-repos-454c2f825e088417.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
```

Comment:
Please also add a second release note that mentions the new container images and repo versions. You can either just add a second file or add a second release note to the same file with this format:
```yaml
---
features:
  - |
    First release note here
  - |
    Second release note here

Reply:
The new release note mentions the rocky rebuild, but also needs the new cloudkitty and neutron tags for centos and ubuntu.

Reply:
Since tag `yoga-20231107T165648` (cloudkitty) and `yoga-20231114T125927` (neutron) are used for all 3 OS in `stackhpc/yoga`, the change was to preserve the use of the old tag for Centos and Ubuntu. There's no newly built cloudkitty or neutron for them.

Reply:
Ah, yes my mistake